proto/aep-api/aep/api/webhook.proto

```diff
@@ -0,0 +1,65 @@
+syntax = "proto3";
+
+package aep.api;
+
+import "google/protobuf/any.proto";
+import "google/protobuf/descriptor.proto";
+import "google/protobuf/timestamp.proto";
+
+option java_package = "dev.aep.api";
+option java_multiple_files = true;
+option java_outer_classname = "NotificationProto";
+option go_package = "aep.dev/api";
+
+// Provides a generic wrapper for all webhook notifications.
+message Notification {
```

Comment:
thoughts on making this compatible with https://github.com/standard-webhooks/standard-webhooks/blob/main/spec/standard-webhooks.md?